files/zh-cn/web/api/web_authentication_api/index.md

```diff
@@ -29,7 +29,7 @@ Web Authentication API（也称作 WebAuthn）使用[asymmetric (public-key) cry
 
 一个典型的注册过程包括如图 1 所示的六个步骤，这些在稍后会进一步描述。这是一个注册过程的概览，所需数据已经被简化。所有的必填字段、可选字段及它们在创建注册请求中的含义可以在 {{domxref("PublicKeyCredentialCreationOptions")}} 字典中找到。类似地，完整的响应字段可以在 {{domxref("PublicKeyCredential")}} 接口（其中 {{domxref("PublicKeyCredential.response")}} 是 {{domxref("AuthenticatorAttestationResponse")}} 的接口）中找到。请注意大多数编写程序的 JavaScript 程序员只会关心第 1 步和第 5 步，分别对应 create() 函数的调用和返回。但是，了解步骤 2 到 4 对于理解在浏览器和认证器中发生了什么以及返回数据的含义至关重要。
 
-![WebAuthn registration component and dataflow diagram](<mdn_webauthn_registration_(r3).png>)
+![WebAuthn registration component and dataflow diagram](mdn_webauthn_registration_r3.png)
```

Comment:
Thanks @OnkarRuikar, as `mdn_webauthn_registration_r3.png` is similar to `webauthn_registration_r4.png` which is used in en-US.

Could we remove the image in zh-CN, and update the filename?

```suggestion
![WebAuthn registration component and dataflow diagram](webauthn_registration_r4.png)
```

Reply:
> Could we remove the image in zh-CN, and update the filename?

Image `mdn_webauthn_authentication_r1.png` is also missing from the page. I've copied r1 and r4 images from the content repo and deleted r3. Now the images match content repo.